ftest.sh

```diff
@@ -144,13 +144,12 @@ fi
 
 trap 'set +e; cleanup' EXIT
 
-CLUSH_ARGS=($CLUSH_ARGS)
-
 DAOS_BASE=${SL_PREFIX%/install}
 if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
     -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
-# allow core files to be generated
+sudo prlimit --pid \$\$ --memlock=\"unlimited\"
```

Comment:
This is only going to set the limit for this shell session which is not the shell session the test are run in, so this has no effect on your test session shells.

Reply:
yes, understood the difference... My brain was spiraling all over the place... I have to revert back to my old method...